### ISSUE TITLE
Support fully automatic redirect

### ISSUE THREAD

**Issue by samie** [mvysny/failover-vaadin#3]
When running read-only application in 2 servers, we want to avoid the pinging altogether since the only thing we can do is either continue using the current server (which is down) or try the other one (which also might be down). 

In the end the result for user is the same, but automatic redirect is at least trying the other server. 

Furthermore, since the application is read-only there is no need to worry about the session getting lost and plain reload is ok.

**Comment by mvysny** [comment]
Thanks! I can see one difference if both servers are down. With proper ping strategy, the reconnect dialog would inform the user that no servers are up and the page will try to reconnect. Without proper ping the browser will be "blindly" redirected to the other server; since it's down a blank page is presented; back button will also show a blank page since the first server is also down.

**Comment by samie** [comment]
Yes, this is true and relevant for not losing the entered information, if the application is not read-only.

**Comment by mvysny** [comment]
I think that losing entered information is more connected to the automatic failover as opposed to manual one (your data may be lost. Continue?).

On the other hand, the ping strategy is more connected to the user experience - the user sees the dialog and understands that the browser is reconnecting on his behalf and he doesn't have to do anything, as opposed to the always-on ping strategy which would simply render blank page, leaving the user to figure out what to do - refresh? Back button?
